### PR TITLE
Triffhir-372

### PR DESCRIPTION
**Pull request by bnorris2**
Allow user to select version of IG Publisher to use when exporting/publishing.

### CODE REVIEW

File: apps/server/src/app/export.controller.ts [hunk 1]
```diff
@@ -182,6 +184,40 @@ export class ExportController extends BaseController {
...
+        const sortedVersions = versions.sort().reverse();
+        const currentVersion = sortedVersions[0];
+        sortedVersions[0] = currentVersion + ' (Current)';
+        return sortedVersions;
+      }).catch(function(reason) {
+        return reason;
+      })
+  }
+
+  private getOptions() {
```
Comment (seanmcilvenna) [inline apps/server/src/app/export.controller.ts:213]: This is not used anywhere else as far as I can tell... would rather just create the options directly in the getPublisherVersions() method.

File: apps/server/src/app/export.controller.ts [hunk 2]
```diff
@@ -182,6 +184,40 @@ export class ExportController extends BaseController {
     }
   }
 
+  /**
+   * The publisherVersion() returns an array of publisher versions from the sonatype.org endpoint defined below.
+   */
+  @Get(':publisher-version')
+  public publisherVersion(): Promise<any> {
```
Comment (seanmcilvenna) [inline apps/server/src/app/export.controller.ts:191]: Consider renaming to getPublisherVersions()

File: apps/server/src/app/export/html.ts [hunk 3]
```diff
@@ -198,7 +201,7 @@ export class HtmlExporter {
     });
   }
 
-  public async export(format: Formats, includeIgPublisherJar: boolean, useLatest: boolean, templateType = 'official', template = 'hl7.fhir.template', templateVersion = 'current'): Promise<void> {
+  public async export(format: Formats, includeIgPublisherJar: boolean, useLatest: boolean, version: string, templateType = 'official', template = 'hl7.fhir.template', templateVersion = 'current'): Promise<void> {
```
Comment (seanmcilvenna) [inline apps/server/src/app/export/html.ts:204]: can remove `useLatest` I believe.

File: apps/server/src/app/export/html.ts [hunk 4]
```diff
@@ -318,7 +321,7 @@ export class HtmlExporter {
 
     this.writePages(this.rootPath);
 
-    this.igPublisherLocation = await this.getIgPublisher(useLatest);
+    this.igPublisherLocation = await this.getIgPublisher(useLatest, version);
```
Comment (seanmcilvenna) [inline apps/server/src/app/export/html.ts:324]: Can remove `useLatest` from `getIgPublisher` as well, I believe.

File: apps/server/src/app/export.controller.ts (hunk 1 above)
Comment (bnorris2) [reply apps/server/src/app/export.controller.ts:213]: agreed

File: apps/server/src/app/export.controller.ts (hunk 2 above)
Comment (bnorris2) [reply apps/server/src/app/export.controller.ts:191]: fixed

**Pull request merged**